specs/gluster_object_detail_views.adoc

```diff
@@ -0,0 +1,198 @@
+// vim: tw=79
+:imagesdir: ./images/
+
+= Tendrl Definitions: Display list of volume and host associated with a gluster cluster discovered by Tendrl
+
+== Problem description
+
+Display list of volume and host associated with a gluster cluster present in Tendrl. 
+
+*Host List*
+
+The details shown for each host are -
+
+* Status
+* Name
+* Role
+* Alerts
+
+The host list should have functionality for 
+
+* filtering by name, role and status.
```

Comment:
So in case of gluster all the node are `gluster/server` whereas in ceph there are two roles applicable for nodes namely `ceph/mon` and `ceph/osd`. At the moment you can have only one role `gluster/server` defined for role based filtering.